<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="LoginBut">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>601</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignUpBut">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>611</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign up </string>
   </property>
  </widget>
  <widget class="QPushButton" name="InfoBut">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>What's RevisionBuddy ?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
